guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/M2MavenRepositoryServiceImplTest.java

```diff
@@ -96,6 +94,31 @@ public static void setupClass() {
                                 "1.0.2.Final");
     }
 
+    @AfterClass
+    public static void tearDown() throws Exception {
+        log.info("Deleting all Repository instances..");
+
+        File dir = new File("repositories");
+        log.info("DELETING test repo: " + dir.getAbsolutePath());
+        deleteDir(dir);
+        log.info("TEST repo was deleted.");
+    }
+
+    public static boolean deleteDir(File dir) {
+        if (dir.isDirectory()) {
```

Comment:
Maybe save some boilerplate by using `FileUtils.deleteDirectory(new File("directory"));` from commons-io which this project already has in its dependencies..